src/pymor/models/basic.py

```diff
@@ -242,6 +278,7 @@ def __init__(self, T, initial_data, operator, rhs, mass=None, time_stepper=None,
                          visualizer=visualizer, cache_region=cache_region, name=name)
         self.T = T
         self.initial_data = initial_data
+        self.linear = operator.linear
```

Comment:
See above. Also we should probably check for linearity of `mass` and `initial_data`.